hack/test_e2e.sh

```diff
@@ -108,20 +108,23 @@ function container_build {
     # build a container image for KPNG                                        #
     #                                                                         #
     # Arguments:                                                              #
-    #   None                                                                  #
+    # arg1: Path for E2E installation dicrectory, or the empty string         #
     ###########################################################################
-    CONTAINER_FILE="Dockerfile"
+    [ $# -eq 1 ]
+    if_error_exit "Wrong number of arguments to ${FUNCNAME[0]}"
+
+    CONTAINER_FILE=${1}
 
     # Running locally it's not necessary to show all info
     QUIET_MODE="--quiet"
     if [ "${ci_mode}" = true ] ; then
         QUIET_MODE=""
     fi
 
-    [ -f "${CONTAINER_FILE}" ]
+    # -f "${CONTAINER_FILE}" ]
```

Comment:
Usually not a good practice to commit commented code :)

Reply:
Specially when it is not supposed to be commented out, I removed the [ instead of the #